packages/neo-one-server-client/src/ServerManager.ts

```diff
@@ -90,6 +92,20 @@ export class ServerManager {
       return undefined;
     }
   }
+  public async getServerVersion(): Promise<string | undefined> {
```

Comment:
This should be used when the client can't connect to the server (presumably because the server is not alive) in `getRelativeServerVersion`

Reply:
(As a style note, let's make `getRelativeServerVersion` a method of this class so that it can access `getServerVersion`)

Reply:
if the server is alive, we immediately check version
https://github.com/neo-one-suite/neo-one/blob/cae10f56d6d09b5926afe96c5f1f7415390509ba/packages/neo-one-server-client/src/ServerManager.ts#L118

Otherwise, if the server is not alive, we start it.

Let me know if there something about this flow that should be changed.

Reply:
👍